edith-step/src/components/ReceiptInput.js

```diff
@@ -87,6 +103,22 @@ export default class ReceiptInput extends React.Component {
       id: Date.now(),
     };
 
+    axios({
+      method: 'post',
+      url: '/user-stats-servlet',
+      data: {
+        itemName: this.state.itemName,
+        itemCategory: this.state.itemCategory,
+        itemPrice: this.state.itemPrice,
+        itemQuantity: this.state.itemQuantity,
+        itemDate: this.getDate(),
+        itemReceiptId: this.state.itemReceiptId,
+        itemCategory: this.state.itemCategory,
+      },
+    }).then((response) => {
+      console.log(response);
```

Comment:
is this .then that only has a print statement in it necessary? could it just use catch

Reply:
I changed it.